fabric_plugin/tunnel.py

```diff
@@ -22,14 +22,11 @@ def documented_contextmanager(func):
 
 
 @documented_contextmanager
-def remote(remote_port, local_port=None, local_host="localhost",
+def remote(local_port, remote_port=0, local_host="localhost",
```

Comment:
tunnel could also be used by users in `run_task` as a general util, in which case, the remote side might not be dynamic as is the case with `CTX_SOCKET_URL`, e.g.. some script that is executed with `fabric.run` will only talk to port 8080.